Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -80,4 +78,18 @@ public CaseDTO updateCase(long suitId, CaseDTO cs) {
 
         return cs;
     }
+
+    public void removeCases(long suitId, List<Long> casesId) {
+        Suit suit = suitDAO.getOne(suitId);
+        suit.getCases().removeIf((c) -> {
```

Comment:
Are you sure suit can not be null?

Reply:
Yes, because we have validation on frontend. Customer need to select suit and select cases that he want tot delete. While he don't do this, the delete button will be disabled.